### DISCUSSION TITLE
Where the module stores the pdf of character sheets?

### DISCUSSION THREAD

**Discussion by Petesuchos** [farling42/fvtt-pdf-pager#87] [Q&A] [answered]
I suspect I might be asking a basic question.
I've uploaded character sheets from PDFs created outside of Foundry. I uploaded the sheets to Foundry VTT. Following the instructions, I temporarily enabled "Read Fields From PDF" so that the data could be loaded.

Now, I want to do the reverse. I'd like to take the character sheets that have undergone several changes since the game began and that are being used in Foundry, and download them for a backup outside the server and to use in tabletop (TT). But I can't figure out where the sheets with the latest changes are stored.

I thought the module would save them over the original PDF sheet that I uploaded, but no, this file remains unchanged.

Any idea where the module might be saving these PDF sheets?

**Comment by farling42** [comment, marked as answer]
The information is stored inside foundry on the individual Actor.

If you display the Actor in the PDF, you can press ctrl-S to open the PDF save dialog, which will download the filled PDF to your local drive.

**Reply by Petesuchos** [reply]
ctrl-S do the trick! Thank you very much!
